docs/root/api/client_features.rst

```diff
@@ -18,3 +18,9 @@ Currently Defined Client Features
   :ref:`overprovisioning_factor<envoy_api_field_ClusterLoadAssignment.Policy.overprovisioning_factor>`
   field. If graceful failover functionality is required, it must be supplied by the management
   server.
+- **envoy.config.supports-resource-in-sotw-response**: This feature indicates that xDS client
```

Comment:
@markdroth how do you see this interacting with `envoy.config.require-any-fields-contain-struct`? Should we spell it out?

Reply:
You're right, we should document that.  I think that `envoy.config.require-any-fields-contain-struct` should apply only to `Any` fields inside of resources, not the ones in the transport protocol itself.  If a client is going to be requesting resources of a particular type, then it already needs to know the proto messages for the transport protocol itself (which would include `Resource` if this client feature is advertised) and for the specific resource type it's requesting.

I think we can just change the wording for `envoy.config.require-any-fields-contain-struct` above to clarify this.

Reply:
Sounds good. LGTM with the docs update ^^.